packages/turf-polygonize/index.js

```diff
@@ -19,6 +18,4 @@ var lineSegment = require('@turf/line-segment');
  * @returns {FeatureCollection<Polygon>} Polygons created
  * @throws {Error} if geoJson is invalid.
  */
-module.exports = function (geojson) {
-    return polygonize(lineSegment(geojson));
-};
+module.exports = polygonize;
```

Comment:
Going to keep the basic function syntax in case anyone wants to expand the TurfJS method they can easily do so.

```javascript
module.exports = function (geojson) {
  return polygonize(geojson);
}
```